pages/common/jmeter.md

```diff
@@ -0,0 +1,20 @@
+# jmeter
+
+> Open source java application designed for load testing functional behavior and measure performance.
+> More information: <https://jmeter.apache.org>.
+
+- Run jmeter in cli mode for a test plan:
+
+`jmeter -n -t {{name}}.jmx`
+    
+- Run jmeter and specify the log output:
+
+`jmeter -n -t {{name}}.jmx -l {{logfile}}.jtl`
```

Comment:
- We prefer long options. There is no fixed convention in our style guide when we do this. But it will be added soon.
- Don't use plain `name` or `logfile` as above. They are paths here so they must be written as:

```suggestion
`jmeter --nogui --testfile {{path/to/file}}.jmx --logfile {{path/to/logfile}}.jtl`
```

```suggestion
`jmeter --nogui --testfile {{path/to/file.jmx}} -l {{path/to/logfile.jtl}}`
```

Reply:
Yep, looks more readable. 
Please validate ✅